catalogue_graph/src/ingestor/models/related_concepts.py

```diff
@@ -55,36 +49,50 @@ def _get_related_concepts(self, key: str) -> list[dict]:
         return related_concepts
 
     @property
-    def fields_of_work(self) -> list[ConceptRelatedTo]:
-        raw_related = self._get_related_concepts("fields_of_work")
-        return transform_related_concepts(raw_related)
+    def fields_of_work(self) -> list[RawNeptuneRelatedConcept]:
+        return [
+            RawNeptuneRelatedConcept(concept)
+            for concept in self._get_related_concepts("fields_of_work")
+        ]
 
     @property
-    def people(self) -> list[ConceptRelatedTo]:
-        raw_related = self._get_related_concepts("people")
-        return transform_related_concepts(raw_related)
```

Comment:
This (and the similar lines) are where RawNeptuneRelatedConcept differed significantly from RawNeptuneConcept.  

I have changed RawNeptuneRelatedConcept so that it behaves in the same manner as RawNeptuneConcept, in that it simply takes a dict representing the concept, and knows where to fish the various raw properties out from.

Transformation now comes later - in the same place as it does for RawNeptureConcept